examples/app_unity_gain_voice/src/app_unity_gain_voice.xc

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2017, XMOS Ltd, All rights reserved
+#include <xs1.h>
+#include <xclib.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>  /* abs */
+#include <math.h>    /* log10 */
+
+#include "src.h"
+
+#define DEBUG_PRINT (0)
+
+#define MAX_ATTEN_DB (-20)
+#define NUM_OF_SAMPLES (100)
+#define MIN_VALUE (1000000)
```

Comment:
I feel that this strategy is a little arbitrary. I would prefer see:
#define NUM_TESTS 32ish
int32_t dc_test_value[NUM_TESTS] = {INT_MIN, INT_MIN+1, INT_MIN+2, INT_MIN-INT_MIN/2, INT_MIN/2, INT_MIN/4, INT_MIN/8, -1024, -128, -1, 0, 1, 128, 1024, ...];
or another good idea is to do a run of random inputs and if there is a failure put it in the above list.

Reply:
If I use low values the attenuation is much higher, for example if I input 100, the upsampling filter returns 87. Is this ok?